zwave_js_server/model/node.py

```diff
@@ -367,10 +367,16 @@ def handle_value_removed(self, event: Event) -> None:
 
     def handle_value_notification(self, event: Event) -> None:
         """Process a node value notification event."""
+        # append metadata if value metadata is available
+        value = self.values.get(get_value_id(self, event.data["args"]))
+        if value and value.data.get("metadata"):
+            event.data["args"]["metadata"] = value.data["metadata"]
```

Comment:
Fine here too, but also something in a future PR where we could maybe forward processing to the value so it can process this event properly.